### DISCUSSION TITLE
MAUI-Nightly no new builds since 16 feb

### DISCUSSION THREAD

**Discussion by Pastajello** [dotnet/maui#20850] [Q&A]
Hello,

for almost 2 weeks I see that maui-nightly has not been updated. The latest version is from 16 feb.
I see new PRs being merged every day, yet even without new build should be triggered. Was nightly turned off?

<img width="1183" alt="image" src="https://github.com/dotnet/maui/assets/28345098/585e6112-1eaf-4013-ab7b-3c0873189777">

**Comment by developer9969** [comment]
Who knows what's going on,guessing nightly build are not stable and they don't want more bugs raised
